fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalCatalog.java

```diff
@@ -125,6 +125,8 @@ public abstract class ExternalCatalog
             CREATE_TIME,
             USE_META_CACHE);
 
+    protected static final int ICEBERG_CATALOG_EXECUTOR_THREAD_NUM = Runtime.getRuntime().availableProcessors();
```

Comment:
Isn't it a bit of a luxury for each catalog to hold as many threads as the number of CPU cores?